app/controllers/users_controller.rb

```diff
@@ -5,6 +5,11 @@ def show
     @orders = Order.where(user_id: current_user.id)
   end
 
+  def total_price(order)
+    order.items.reduce(0) {|sum, item| sum + item.price}
```

Comment:
Prefer `inject` over `reduce`.<br>Space between { and | missing.<br>Space missing inside }.